src/main/java/uk/gov/justice/digital/hmpps/whereabouts/model/Court.kt

```diff
@@ -9,11 +9,11 @@ import javax.persistence.Table
 @Entity
 @Table(name = "enabled_court")
 @ApiModel(description = "Video Link Booking related information for a court")
-data class Court(
```

Comment:
Don't use data class for entities and don't use immutable properties they don't work well with Hibernate.
In this case equals and hash code don't need overriding because these objects aren't used in sets.